test/counter.spec.ts

```diff
@@ -1,17 +1,13 @@
 // Copyright (C) 2017-2019 Brainbean Apps OU (https://brainbeanapps.com).
 // License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
-import 'mocha'
-import chai from 'chai'
-import chaiAsPromised from 'chai-as-promised'
+import 'ts-jest'
 import { Promise as BluebirdPromise } from 'bluebird'
 import { PassThrough } from 'stream'
 import Counter from '../src/counter'
-import * as logger from 'winston'
+import logger from '../src/logger'
```

Comment:
for tests use separate logger

Reply:
we don't use logger here at all, it imports here to switch off input in console

Reply:
then just import winston and disable it directly or configure in otherwise - within test